README.rst

```diff
@@ -135,39 +125,31 @@ For more details and alternative ways to install, see the
 
 .. _Installation section: http://pywbem.readthedocs.io/en/stable/intro.html#installation
 
-Usage
------
-
-For information on how to use pywbem, or how to contribute to it, go to the
-`pywbem documentation`_ .
-
-.. _pywbem documentation: http://pywbem.readthedocs.io/en/stable/
-
 Documentation
 -------------
 
 The lastest pywbem documentation is available on ReadTheDocs:
 
-* `Documentation for latest version on Pypi`_
+* `Documentation for latest released version`_
 
-.. _Documentation for latest version on Pypi: http://pywbem.readthedocs.io/en/stable/
+.. _Documentation for latest released version: http://pywbem.readthedocs.io/en/stable/
 
-* `Documentation for master branch in the Git repo`_
+* `Documentation for latest development version`_
```

Comment:
suggest change to latest unreleased development version

Reply:
DONE